ENIGMAsystem/SHELL/Universal_System/Extensions/Alarms/alarmcode.cpp

```diff
@@ -7,8 +7,6 @@
 #include "implement.h"
 #include "include.h"
 
-extern bool argument_relative;
```

Comment:
This has broken the fps example by Mark Overmars:

```
C:/Users/Owner/AppData/Local/ENIGMA/Preprocessor_Environment_Editable/IDE_EDIT_objectfunctionality.h:219:5: error: reference to 'argument_relative' is ambiguous
     argument_relative = 0;
     ^
```

`argument_relative` should be a built-in global:
https://forum.yoyogames.com/index.php?threads/argument_relative.13530/